subsys/mgmt/mcumgr/smp_bt.c

```diff
@@ -41,6 +55,94 @@ static struct bt_uuid_128 smp_bt_svc_uuid = BT_UUID_INIT_128(
 static struct bt_uuid_128 smp_bt_chr_uuid = BT_UUID_INIT_128(
 	BT_UUID_128_ENCODE(0xda2e7828, 0xfbce, 0x4e01, 0xae9e, 0x261174997c48));
 
+/* Helper function that allocates conn_param_data for a conn. */
+static struct conn_param_data *alloc_conn_param_data(struct bt_conn *conn)
```

Comment:
Checked that and it would be problematic to have. Having this argument `const` triggers compilation warning when assigning this pointer to non const `conn_data[i].conn`. I could add const to structure member `conn` but then, `bt_conn_*` API is used inconsistent. I will leave this as is.